cointop/config.go

```diff
@@ -262,22 +237,12 @@ func (ct *Cointop) ConfigToToml() ([]byte, error) {
 	})
 	portfolioIfc["holdings"] = holdingsIfc
 
-	var columnsIfc interface{} = ct.State.portfolioTableColumns
-	portfolioIfc["columns"] = columnsIfc
-
-	var currencyIfc interface{} = ct.State.currencyConversion
-	var defaultViewIfc interface{} = ct.State.defaultView
-	var defaultChartRangeIfc interface{} = ct.State.defaultChartRange
-	var colorschemeIfc interface{} = ct.colorschemeName
-	var refreshRateIfc interface{} = uint(ct.State.refreshRate.Seconds())
-	var cacheDirIfc interface{} = ct.State.cacheDir
+	portfolioIfc["columns"] = ct.State.portfolioTableColumns
```

Comment:
can `portfolioIfc` be constructed similar to `favoritesMapIfc` or `tableMapIfc`?

Reply:
done